samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -39,8 +41,10 @@
 
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Information"
+      "Default": "Warning",
+      "Microsoft.AspNetCore.Hosting": "Information",
```

Comment:
I found this combination to be the sweet spot (gives us all we want but doesn't fill half a screen on each request

Reply:
I would recommend Default being "Information" since this is where any additional code added by developers will show up.

Reply:
We  can stick with this for now, though.